packages/fuels-accounts/src/kms.rs

```diff
@@ -0,0 +1,3 @@
+mod aws;
```

Comment:
Why nest it this deep? I'd inline it and have `client` and `wallet` on this level.

Reply:
i'll add google kms support so i thought it would be better to separate them.